karma.conf.js

```diff
@@ -103,39 +54,30 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha', 'sinon', 'chai', 'sinon-chai'],
-
+    frameworks: ['requirejs', 'mocha', 'chai'],
```

Comment:
Fair enough. The diff for the tests didn't look too big though so it might have been worth it.